docs/orchestration/global-hooks/guides.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Global Hooks Guide
+sidebarTitle: "Guide"
+---
+<img
+  alt="Penguin Fishing"
+  src="https://github.com/mage-ai/assets/blob/main/penguin_fishing.gif?raw=True"
+/>
+Follow this step by step guide on creating a global hook in Mage.
+
+### <b>Step 1: Navigate to the Global Hooks UI</b>
+1. Log in to your Mage account.
+1. From the left-hand sidebar, click on "Global Hooks" under the "Settings" section.
+1. Click on the "+ Add new global hook" button.
+
+### <b>Step 2: Configure the Hook</b>
+1. **Hook UUID**: Enter a unique name for your hook.
+2. **Resource Type**: Select the type of resource (API call type) from the resource drop down.
+3. **Operation Type**: Select the type of operation needed for your hook.
+4. **Targeting Conditions**: Enter conditions if you want it to apply the hook to only certain pipelines. Leave this field blank to apply the hook to all pipelines. 
+5. **Before Operation Starts Toggle**: Toggle to on if you want the hook to run before an operation starts.
+6. **After Operation Completes Toggle**: Toggle to on if you want the hook to run after an operation ends.
+7. **Run if Operation Succeeds Toggle**: Toggle to on if you want to hook to run only if the operation succeeds.
+8. **Run if Operation Fails Toggle**: Toggle to on if you want the hook to run if the operation fails.
+9. **Pipeline to Execute**: Select the pipeline you want to execute the hook from.
+10. **Valid Code Snapshot**: Click the "Snapshot" button to validate the pipeline code.
```

Comment:
By "Snapshot" button, do you mean the "Create snapshot of code" button? We should be explicit in what button it is. If we wanted to abbreviate that, "Create snapshot" button would make more sense to me.

Reply:
Added "Create snapshot of code" and tested in mintlify dev.
<img width="793" alt="Screenshot 2024-06-19 at 12 20 24 PM" src="https://github.com/mage-ai/mage-ai/assets/92221031/05fd3863-bb74-4d3f-ba4e-d9e717789dbc">